zipkin-storage/mysql-v1/src/main/java/zipkin2/storage/mysql/v1/internal/generated/DefaultCatalog.java

```diff
@@ -4,40 +4,30 @@
 package zipkin2.storage.mysql.v1.internal.generated;
 
 
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-import javax.annotation.Generated;
-
 import org.jooq.Schema;
 import org.jooq.impl.CatalogImpl;
 
 
 /**
  * This class is generated by jOOQ.
  */
-@Generated(
```

Comment:
these changes happened via the mysql README which has a command to run when changing jooq versions